doc/nrf/ug_avsystem.rst

```diff
@@ -0,0 +1,202 @@
+.. _ug_avsystem:
+
+AVSystem integration
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+AVSystem is a software solution provider that provides a device management platform.
+This platform is based on the LwM2M protocol with |NCS| compatibility for Nordic Semiconductor's cellular IoT products.
+It handles the entire process from onboarding till decommissioning of devices and its integration with |NCS| ensures secure and power efficient device management solutions using Nordic Semiconductor's cellular devices.
```

Comment:
Should we split it into two sentences and add a blank line to improve the readability?
```suggestion
It handles the entire process from onboarding till decommissioning of devices.

Its integration with |NCS| ensures secure and power efficient device management solutions using Nordic Semiconductor's cellular devices.
```